addons/hr_holidays/static/src/js/time_off_calendar.js

```diff
@@ -94,8 +94,8 @@ odoo.define('hr_holidays.dashboard.view_custo', function(require) {
                     res_model: "hr.leave",
                     view_id: ids,
                     context: {
-                        'default_date_from': moment().format('YYYY-MM-DD'),
-                        'default_date_to': moment().add(1, 'days').format('YYYY-MM-DD'),
+                        'default_date_from': moment().format('YYYY-MM-DD 07:00:00'),
+                        'default_date_to': moment().format('YYYY-MM-DD 19:00:00'),
```

Comment:
Picking two dates on the calendar has a default start and end time of 7AM and 7PM...